src/librustdoc/clean/blanket_impl.rs

```diff
@@ -27,7 +28,7 @@ impl<'a, 'tcx> BlanketImplFinder<'a, 'tcx> {
 
         debug!("get_blanket_impls({:?})", ty);
         let mut impls = Vec::new();
-        for &trait_def_id in self.cx.all_traits.iter() {
+        for &trait_def_id in self.cx.tcx.all_traits(LOCAL_CRATE).iter() {
```

Comment:
I wonder if this won't remove some auto impl discovering...

Reply:
This is (AFAICT) equivalent because queries once evaluated are always the same, and in this case we don't touch the all_traits vector except on creation.

Reply:
Well, if tests pass then I guess nothing changed in the trait discovery so that's all good.